### DISCUSSION TITLE
2011/08/03/packed-project-libraries

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#342] [Announcements]
# 2011/08/03/packed-project-libraries

全书： https://lv.qizhen.xyz/

https://blog.qizhen.xyz/2011/08/03/packed-project-libraries

**Comment by ruanqizhen** [comment]
之前博客下的留言：

北京回收拉菲
2011年08月8日 23:04 编辑
是否在项目里使用它还要看其他方面的功能。

回复
liumc
2011年08月5日 00:09 编辑
Packed Project Libraries 感觉的确是个好的东西，不过我在lavag看到有贴说，2010版本的Packed Project Libraries 性能不好，而且有时候不是很方便，在新发布的2011相关资料里面也没看到相关的改进说明。

回复
ruanqizhen
2011年08月5日 09:42 编辑
我还没有在实际项目中使用过Packed Project Libraries，因此没有做过性能测试。但是LabVIEW的启动画面在使用了Packed Project Libraries以后，装载速度快多了，说明它至少在某些方面是可以提高程序性能的。
不过性能只是一方面，是否在项目里使用它还要看其他方面的功能。

回复
